Seasons Defense/Assets/Scripts/EnemyMovement.cs

```diff
@@ -0,0 +1,47 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class EnemyMovement : MonoBehaviour
+{
+    //The speed of the enemies
+    public float speed = 5f;
+
+    //To be able to switch direction from left to right or right to left
+    private bool switchMovement = true;
```

Comment:
Can we change this to something along these lines (since the bool is being checked for the direction to travel):
```suggestion
    private bool _moveRight= true;
```